### PR TITLE
Automaticly validate expenses when paying payslip

### PR DESCRIPTION
**Pull request by open-net-sarl**

### CODE REVIEW

File: l10n_ch_hr_payroll/models/hr_payslip.py [hunk 1]
```diff
@@ -139,3 +139,12 @@ def compute_sheet(self):
...
 
         return ret
+
+    def process_sheet(self):
+        ExpenseObj = self.env['hr.expense']
+        expenses = ExpenseObj.search([
+            ('slip_id', '=', self.id)
+        ])
+        for expense in expenses:
+        expenses.state = 'done'
```
Comment (yvaucher) [inline l10n_ch_hr_payroll/models/hr_payslip.py]: You have an indentation error here

And `s/expenses/expense/`

**Comment by yvaucher** [comment]
Could you change the commit author ? You might not want to give the credits to `root`

**Comment by yvaucher** [comment]
@open-net-sarl What about this one ? Could you fix it ?

**Comment by open-net-sarl** [comment]
@yvaucher A customer has noticed a potential issue when move_id is None.
Can you merge this PR quickly ? I fixed the travis functional test.
Thanks

**Comment by yvaucher** [comment]
@open-net-sarl author on first commit is still wrong and you have a lint issue.

I merge squashing the commits to fix author and fixing the 2 pep8 issues

**Comment by yvaucher** [comment]
https://github.com/OCA/l10n-switzerland/commit/7054cc95059393fc5670a5ff73201023b6db8c72

**Pull request closed without merging**